.rubocop.yml

```diff
@@ -41,6 +42,10 @@ Metrics/MethodLength:
     - 'hash'
   Max: 15 # default 10
 
+Metrics/ModuleLength:
+  Exclude:
+    - spec/support/work_mapping_fixtures.rb # it's meant to be a large container of mappings
```

Comment:
Since we intend to skip this Rubocop violation, move it from an inline skip to a configured skip with a rationale.